src/main/resources/archetype-resources/messages/pom.xml

```diff
@@ -2,47 +2,39 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <parent>
-        <groupId>${groupId}</groupId>
         <artifactId>${rootArtifactId}</artifactId>
+        <groupId>${groupId}</groupId>

```

Comment:
Blame the pomsorter from intellij. I just figured out that this plugin does some kind of custom sorting because the official https://maven.apache.org/developers/conventions/code.html (POM Code Convention) voted for an other sort order.

You can "test" or preview this kind of ordering with:
`mvn com.github.ekryd.sortpom:sortpom-maven-plugin:sort -Dsort.pomFile="src/main/resources/archetype-resources/silo/pom.xml" -Dsort.predefinedSortOrder="recommended_2008_06"` out of the root directory of this repository.

Let me know your thoughts.